iroha_client/src/client.rs

```diff
@@ -548,7 +544,7 @@ pub mod uri {
     /// The URI used for block synchronization.
     pub const BLOCK_SYNC: &str = "/block";
     /// The web socket uri used to subscribe to pipeline and data events.
-    pub const SUBSCRIPTION: &str = "/events";
-    /// Get pending transactions on leader.
-    pub const PENDING_TRANSACTIONS_ON_LEADER: &str = "/pending_transactions_on_leader";
+    pub const SUBSCRIPTION_URI: &str = "/events";
+    /// Get pending transactions.
+    pub const PENDING_TRANSACTIONS_URI: &str = "/pending_transactions";
```

Comment:
Could you also remove `uri` suffix from here

Reply:
Removed.